pkg/cluster/check/etcd.go

```diff
@@ -95,3 +133,24 @@ func sliceCollect[S ~[]V, V any, R any](slc S, fn func(V) R) []R {
 
 	return r
 }
+
+func containsAll[S ~[]V, V comparable](input S, check S) bool {
```

Comment:
nit: it might be more descriptive to use set semantics here?

(and probably arguments should be swapped, subset << set), this is easier than to understand `containsAll` - what should contain what? :)

```suggestion
func isSubset[S ~[]V, V comparable](set S, subset S) bool {
```